gui.py

```diff
@@ -157,27 +164,21 @@ def getDescriptionString():
         elif searchType == -4:
             return str(filter_input.text())
 
-    def showTitleInCentre():
+    def show_title_in_centre():
         return
         print('Added')
         item = graphics_view.item(graphics_view.count() - 1)
-        toShow = '<b>' + getDescriptionString() + '</b>\n' + str(item.text())
+        toShow = '<b>' + get_description_string() + '</b>\n' + str(item.text())
         print(toShow)
         item.setText(toShow)
 
     graphics_view.model().rowsInserted.connect(showTitleInCentre)
+    """
```

Comment:
@owenl131 This still needed?

Reply:
This one is for the showing the original search query in the middle pane. But doesn't work yet because the UI doesnt update immediately for some reason when I setText

Reply:
Orkei. Then we just merge with this commented out?

Reply:
ok